app/views/helpers/businessAddressHelper.scala.html

```diff
@@ -0,0 +1,32 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.BusinessListResponseModel
+
+@(model: BusinessModel)(implicit request: Request[_], messages: Messages, appConfig: config.FrontendAppConfig, user: auth.MtdItUser[_])
+
+@defining(List(model.businessAddressLineOne, model.businessAddressLineTwo, model.businessAddressLineThree, model.businessAddressLineFour, model.businessPostcode).flatten) { bizList =>
+
+    @bizList.zipWithIndex.map {
```

Comment:
Instead of using a Table can you use the styling from the Prototype. It uses:

```
<dl class="govuk-check-your-answers cya-questions-short tax-estimate">
        <div>
          <dt class="cya-question">
            Trading name
          </dt>
          <dd class="cya-answer">
              ........
          </dd>          
        </div>        

        <div>
          <dt class="cya-question">
            Business address
          </dt>
          <dd class="cya-answer">
                 ....
          </dd>
        </div>
      </dl>
```

This supports mobile devices better